waiter/src/waiter/scheduler/shell.clj

```diff
@@ -83,6 +83,9 @@
     (doseq [[env-key env-val] (seq environment)]
       (when-not (nil? env-val)
         (.put process-env env-key env-val)))
+    ;; TODO maybe some day this will be WAITER_SANDBOX
+    (when-not (contains? process-env "MESOS_SANDBOX")
+      (.put process-env "MESOS_SANDBOX" (.getAbsolutePath working-dir)))
```

Comment:
For consistency, we need `MESOS_SANDBOX` to represent the working directory. This name violation has already leaked into the Kubernetes implementation.